code/game/objects/structures/transit_tubes/transit_tube_pod.dm

```diff
@@ -11,7 +11,7 @@
 
 /obj/structure/transit_tube_pod/New(loc)
 	..()
-	air_contents.assert_gases("o2", "n2")
+	air_contents.add_gases("o2", "n2")
```

Comment:
Why this change, also port to Initialize please

Reply:
`assert_gases` is just `add_gases` with a condition to return early if the gas already exists.

But we know the gas does not exist yet, so using `add_gases` here is slightly more idiomatic and also a tiny micro-optimization.